examples/docusaurus-2/README.md

```diff
@@ -0,0 +1,33 @@
+# Website
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
+
+```
+$ yarn start
+```
+
+This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
```

Comment:
Does it really open a browser window? I didn't think we supported that due to the proxied dev server address

Reply:
I didn't modify any of the example code, I just created it through their CLIs. And that's with `yarn start`.